login-workflow/src/contexts/OktaAuthContext/i18nOktaAuthInstance.ts

```diff
@@ -0,0 +1,84 @@
+import i18next from 'i18next';
+import { AuthDictionaries } from '../AuthContext/AuthDictionaries';
+import { SharedDictionaries } from '../SharedDictionaries';
+import { OktaAuthDictionaries } from './OktaAuthDictionaries';
+
+export const i18nOktaAuthInstance = i18next.createInstance(
```

Comment:
Is it possible to continue to use the Auth dictionaires and share them between Okta/Custom? Adding a new set of dictionaries like this means we would need to go and update all of our I18N instructions as well. Probably easier to just add one extra string to the existing dictionary.

Reply:
This was updated and removed in latest commit. Locally I don't see this file but for some reason its here in the PR....